homeassistant/components/overkiz/cover_entities/awning.py

```diff
@@ -67,3 +72,43 @@ async def async_open_cover(self, **kwargs: Any) -> None:
     async def async_close_cover(self, **kwargs: Any) -> None:
         """Close the cover."""
         await self.executor.async_execute_command(OverkizCommand.UNDEPLOY)
+
+    @property
+    def is_opening(self) -> bool | None:
+        """Return if the cover is opening or not."""
+
+        if self.assumed_state:
+            return None
```

Comment:
Can we double check if we need this check? I would say that core would need to have a check like this, since `assumed_state` (state-less) would not know when it is opening?

Reply:
I agree, it should be done by the Core directly. And as all my devices are statefull, I cannot easily test the behavior.